docs/samples/v1beta1/torchserve/v2/bert/bert_gpu.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: "torchserve-bert-v2"
+spec:
+  predictor:
+    pytorch:
+      protocolVersion: v2
+      storageUri: gs://kfserving-examples/models/torchserve/huggingface
```

Comment:
use `gs://kfserving-examples/models/torchserve/bert/v2`

Reply:
I used the same uri used in [bert.yaml](https://github.com/kserve/kserve/blob/48146646d76a133da09bf65f1c84f69df9e2a0e6/docs/samples/v1beta1/torchserve/v2/bert/bert.yaml#L9) . `bert.yaml` and `bert_gpu.yaml` will use the same uri.

Reply:
yes please use `gs://kfserving-examples/models/torchserve/bert/v2`